catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -364,7 +371,8 @@
                             versioning-common,
                             tika-core,
                             failsafe,
-                            solr-dependencies
+                            solr-dependencies,
+                            boon
```

Comment:
Boon is a bundle, you shouldn't need to embed this.

Reply:
removed